app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.scss

```diff
@@ -0,0 +1,101 @@
+.list-view-pf-additional-info-item {
+  .mappings-expand-label-group {
+    display: inline-block;
+  }
+  .mappings-expand-label {
+    line-height: 16px;
+    text-align: left;
+  }
+}
+
+.infra-mappings-list-view {
+  .list-view-pf-description {
+    width: 30%;
+
+    .list-group-item-heading {
+      padding-left: 30px;
+      margin-bottom: 0px;
+    }
+    .list-group-item-text {
+      padding-left: 30px;
+      padding-top: 3px;
+    }
+  }
+  @media (min-width: 992px) {
+    .list-view-pf-description {
+      width: 30%;
+    }
+  }
+  .list-view-pf-additional-info-item {
+    .fa,
+    .pficon {
+      font-size: 19px;
+    }
+    strong {
+      font-size: 13px;
+    }
+    .list-view-pf-expand {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  // custom mapping styles - match mapping row heights
+  // and show borders around mapping row groups
+  .infra-mapping-networks-row,
+  .infra-mapping-datastores-row,
+  .infra-mapping-clusters-row {
+    display: flex;
+    flex-wrap: wrap;
+    & > [class*='col-'] {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 10px;
+      margin-top: 10px;
+      &:nth-child(odd) {
+        padding-right: 0px;
+      }
+      &:nth-child(even) {
+        padding-left: 0px;
+      }
+    }
+    .arrow-wrapper {
+      width: 25%;
+      height: 30px;
+      margin-top: 14px;
+    }
+    .arrow-stem {
+      background: transparent;
+      width: calc(100% - 15px);
+      height: 5px;
+      border-bottom: 1.5px solid black;
+      margin-top: 3px;
+    }
+    .arrow-head {
+      border-top: 8px solid transparent;
+      border-left: 12px solid black;
+      border-bottom: 8px solid transparent;
+    }
+    .arrow-wrapper,
+    .arrow-stem,
+    .arrow-head {
+      float: left;
+    }
+  }
+  .infra-mapping-source-container {
+    display: flex;
+    flex-direction: row;
+  }
+  .infra-mapping-item-source,
+  .infra-mapping-item-target {
+    height: 100%;
+    padding: 10px;
+    width: 75%;
+  }
+  .infra-mapping-item-source {
+    border: 1px solid black;
```

Comment:
Though I wasn't asked, we'll likely wanta use something from https://github.com/patternfly/patternfly/blob/master/src/less/color-variables.less#L76 for this color

Reply:
yep... colors t.b.d...thanks!!

Reply:
Something like... 

``` @import './node_modules/patternfly/dist/sass/patternfly/color-variables';```

then `$color-pf-blue`

I can't for the life of me understand why its so difficult to get to the colors though 🤔 😢

Reply:
i used `$color-pf-blue-400` after eyeballing this a few times... it was the darkest pf-blue color I could denote any distinction. The other colors do not differentiate enough from the the left-hand black border w/ these 👀 . Please note that... we will need to keep a patternfly or patternfly-react dependency here in v2v plugin to get those imported sass variables in this plugin (from the npm installed package). So my vote is to just keep patternfly-react since it depends on patternfly and we going to continue consuming es6 helpers from PF React in the future)...

HOWEVER - patternfly-react (and other ui-classic common packages) should also still be updated in ui-classic when doing upgrades from now on...

cc: @himdel 
https://github.com/ManageIQ/guides/pull/314

another twist on this would be... w/ PF Next (and PF React Next components coming) - we'll be exporting the CSS directly in the ES6 module alongside the React component (so that problem kinda dissipates). More on this in a PR coming upstream soon....but it shouldn't affect us until after GA (when we could even consider PF next components).

lots of tangents but figured it would be good to note this here 🍰 

📚